### DISCUSSION TITLE
Lluvia de ideas

### DISCUSSION THREAD

**Discussion by victorcespedes864** [zrock-dev/Project_X#48] [Ideas]
pongan sus ideas que se puede hacer con nuestro cine , no piensen mucho en como hacerlo o si tenems los recursos necesarios para hacerlo

**Comment by victorcespedes864** [comment]
una manera de modificar los descuentos , agregarlos o modificarlos

**Comment by victorcespedes864** [comment]
imprimir ua factura

**Comment by zrock-dev** [comment]
Que podamos interactuar con el codigo desde la interfaz.

**Comment by zrock-dev** [comment]
Que los clientes registrados tengan una tarjeta donde se guarde su informacion y la cantidad de puntos.

**Comment by zrock-dev** [comment]
Implementar un sistema para comprar souvenirs y comida chatarra
Y que te imprima el icono de lo que has comprado.
![image](https://user-images.githubusercontent.com/64939916/159206039-5bcdcaf8-35e3-4350-925b-c4944ea14eee.png)

**Comment by zrock-dev** [comment]
Que la boleteria nos muestre el horario y las peliculas de la semana (Una cartelera),  junto con las salas donde se van reproducir, y el aforo.

**Reply by zrock-dev** [reply]
Interaccion por consola.

**Comment by zrock-dev** [comment]
Que la sala de cine tenga una seccion para añadir comentarios sobre la pelicula y darle una calificacion.

**Comment by zrock-dev** [comment]
Cambiar todo el idioma del codigo a ingles para practicar.

**Comment by zrock-dev** [comment]
Implementar un sistema de tiempo. Y que basado en este las Salas vayan cambiando de funcion.

**Comment by TheLocoto** [comment]
Mejorar el candyBar, mostrando lo que se puede comprar ahi

**Comment by TheLocoto** [comment]
Analizar y hacer uso de interfaces y herencias

**Reply by zrock-dev** [reply]
Lo comentaria de acuerdo a su avance en el codigo

**Comment by victorcespedes864** [comment]
comprar boletos para peliculas proximas a estrenarse y que no se estrenaron todavia

**Comment by zrock-dev** [comment]
Hay que limpiar el codigo para hacer ciertas cosas estaticas.
Hay que limpiar la info de los JUnit para que vaya acorde con la base de datos.
